### PR TITLE
Issue #1386 Impose force for delete during unregistration process

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
Code looks fine to me. I will do a round of test and then merge it.

File: cmd/minishift/cmd/profile/profile_delete.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func runProfileDelete(cmd *cobra.Command, args []string) {
 	if !exists {
 		fmt.Println(fmt.Sprintf("VM for profile '%s' does not exist", profileName))
 	} else {
-		registrationUtil.UnregisterHost(api, false)
+		registrationUtil.UnregisterHost(api, false, force)
```
Comment (gbraad) [inline cmd/minishift/cmd/profile/profile_delete.go]: can this named the same as the previous used in `delete.go` ?

File: cmd/minishift/cmd/registration/util.go [hunk 2]
```diff
@@ -55,7 +55,7 @@ func RegisterHost(libMachineClient *libmachine.Client) {
 	}
 }
 
-func UnregisterHost(api libmachine.API, allowToSkipUnregistration bool) {
+func UnregisterHost(api libmachine.API, allowToSkipUnregistration bool, force bool) {
```
Comment (gbraad) [inline cmd/minishift/cmd/registration/util.go]: can a full description be used for the variable, pretty much like `allowToSkipUnregistration`? eg. `forceDeletion` ?

**Comment by praveenkumar** [comment]
@gbraad updated as per your suggestions.

**Pull request merged**